secure/storage/vault/Cargo.toml

```diff
@@ -14,11 +14,11 @@ base64 = "0.12.3"
 chrono = "0.4.15"
 once_cell = "1.4.1"
 proptest = { version = "0.10.1", optional = true }
-rustls = "0.17.0"
+native-tls = "0.2.4"
```

Comment:
damn, we should give feedback to these folks that they need to introduce wrapper types